RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/Program.cs

```diff
@@ -0,0 +1,292 @@
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputPlaceInShelf(shelf);
+            }
+
+        }
+        public static CShelf buildShelfs()
+        {
+            CShelf shelf = new CShelf();
+            inputnumberLevel(shelf);
+            inputPlaceInShelf(shelf);
+            Console.WriteLine(shelf.ToString());
+            return shelf;
+        }
+
+        public static void inputModel(CRefrigerator refrigerator)
+        {
+            Console.WriteLine("please enter model");
+            try
+            {
+                refrigerator.Model = Console.ReadLine();
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputModel(refrigerator);
+            }
+
+        }
+        public static void inputColor(CRefrigerator refrigerator)
+        {
+            Console.WriteLine("please enter color");
+            try
+            {
+                refrigerator.Color = Console.ReadLine();
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputColor(refrigerator);
+            }
+
+        }
+        public static void inputNumberOfShelves(CRefrigerator refrigerator)
+        {
+            Console.WriteLine("please enter NumberOfShelves");
+            try
+            {
+                refrigerator.NumberOfShelves = int.Parse(Console.ReadLine());
+            }
+            catch (Exception e)
+            {
+
+                Console.WriteLine(e.Message);
+                inputNumberOfShelves(refrigerator);
+            }
+
+        }
+
+        public static CRefrigerator buildRefrigerator()
+        {
+
+            CRefrigerator refrigerator = new CRefrigerator();
+            inputModel(refrigerator);
+            inputColor(refrigerator);
+            inputNumberOfShelves(refrigerator);
+
+            return refrigerator;
+        }
+
+        static void Main(string[] args)
+        {
+            bool flag = true;
+            //items
+
+            CItem item1 = new CItem();
+            item1 = buildItem();
+            CItem item2 = new CItem();
+            item2 = buildItem();
+
+            //shelfs
+            CShelf shelf1 = new CShelf();
+            shelf1 = buildShelfs();
+            shelf1.AddItem(item1);
+            shelf1.AddItem(item2);
+
+            //refrigerators
+            List<CRefrigerator> refrigerators = new List<CRefrigerator>();
+            CRefrigerator refrigerator = new CRefrigerator();
+            refrigerator = buildRefrigerator();
+            refrigerator.Shelfs.Add(shelf1);
+            refrigerators.Add(refrigerator);
+            Console.WriteLine("The Refrigerator app ❤ ");
+
+            while (flag)
+            {
+                Console.WriteLine("Press 1: the program will print all the items on the refrigerator and all its contents.");
```

Comment:
Click or Press?

Reply:
fixed
Sorry for the mistake of not paying attention
and lack of time